docker/dex/config.yaml

```
@@ -0,0 +1,138 @@
+    # Some providers require passing client_secret via POST parameters instead
+    # of basic auth, despite the OAuth2 RFC discouraging it. Many of these
+    # cases are caught internally, but some may need to uncomment the
+    # following field.
+    #
+    # basicAuthUnsupported: true
+    
+    # List of additional scopes to request in token response
+    # Default is profile and email
+    # Full list at https://dexidp.io/docs/custom-scopes-claims-clients/
+    scopes:
+     - openid
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 5

(indentation)